src/main/assembly/dist/res/template/License.html

```diff
@@ -0,0 +1,101 @@
+                background-size: 100px 30px;
+            }
+            @bottom-center {
+                content: "$FooterText - page " counter(page) "/" counter(pages);
+                font: 14px "Bitstream Charter";
+            }
+        }
+        body {
+            font: 14px "Bitstream Charter";
+            text-align: justify;
+        }
+        ol li ul {
+            list-style-type: disc;
+        }
+        ol {
+            list-style-type: lower-alpha;
+        }
+        table {
+            border-collapse: collapse;
+            border-spacing: 0;
+            display: table;
+            border: 1px solid black;
+            margin-left: auto;
+            margin-right: auto;
+        }
+        tr {
+            border-bottom: 1px solid black;
+        }
+        table td, table th {
+            padding: 6px 8px;
+            display: table-cell;
+            vertical-align: top;
+            border-right: 1px solid black;
+        }
+        .error {
+            color: red;
+            font-size: 150%;
+        }
+    </style>
+</head>
+<body>
+
+#parse("Body.html")
+
+#if ($OpenAccess)
+    <p><strong>[Open Access: unlimited access without registration of user registration]</strong></p>
+    <p>The Depositor agrees to the dataset being made available in accordance with the conditions of the Creative Commons Zero Waiver, the CC0 1.0 Universal Public Domain Dedication (Appendix 1). In doing so, the Depositor renounces all possible rights relating to the dataset.</p>
+#elseif ($OpenAccessForRegisteredUsers)
+    <p><strong>[Open Access for Registered Users: unlimited access for registered users]</strong></p>
+    <p>The Repository is permitted to make the dataset available to all persons, legal entities and organisations registered with the Repository.</p>
+#elseif ($OtherAccess)
+    <p><strong>[Other Access: the data are not available via EASY]</strong></p>
+    <p>The dataset will be made available by means of another method to be agreed with the Repository.</p>
+#elseif ($RestrictGroup)
+    <p><strong>[Restricted Access: access restricted to registered persons or group members, N.B. only for archeology]</strong></p>
+    <p>The Depositor may grant access permission in advance for persons, legal entities and organisations that belong to one of the user groups specified by DANS and/or the Depositor.</p>
+#elseif ($RestrictRequest)
+    <p><strong>[Restricted Access: access with the permission of the Repository]</strong></p>
+    <p>The Repository is permitted to make the dataset available to persons, legal entities and organisations registered with the Depositor only after receiving express permission from the Depositor.<p>
+#else
+    <!-- default access right
+         never expect to be in this case, but I can't find an error function in Velocity
+         so I use a error message in the pdf itself -->
+    <p class="error"><strong>NO VALID VALUE FOR THE ACCESS CATEGORY FOUND!!!</strong></p>
```

Comment:
You are using a class here, why using also strong? Rather define bold in the css.